src/renderer/src/pages/game-details/modals/game-options-modal.tsx

```diff
@@ -116,9 +117,26 @@ export function GameOptionsModal({
     updateGame();
   };
 
+  const handleChangeLaunchOptions = async (event) => {
+    const value = event.target.value;
+
+    setLaunchOptions(value);
+
+    window.electron.updateLaunchOptions(game.id, value).then(updateGame);
+  };
+
+  const handleClearLaunchOptions = async () => {
+    setLaunchOptions("");
+
+    window.electron.updateLaunchOptions(game.id, null).then(updateGame);
+  };
+
```

Comment:
maybe a debounce would be good? 
if you spam this text input you can see that the cpu use raises a bit because it is calling 2 queries in sequence every input change

Reply:
It really affects performance, I implemented a 1sec debounce like you suggested